next/components/forms/widget-wrappers/BAObjectFieldTemplate.tsx

```diff
@@ -20,58 +39,71 @@ const getPropertySpacing = (isInColumnObject: boolean, isFirst: boolean, isLast:
   }
 }
 
+const ColumnDisplay = ({
+  uiOptions,
+  children,
+}: PropsWithChildren<{ uiOptions: ObjectFieldUiOptions }>) => {
+  if (uiOptions.columns && uiOptions.columnsRatio) {
+    const gridTemplateColumns = uiOptions.columnsRatio
+      .split('/')
+      .map((value) => `minmax(0, ${value}fr)`)
+      .join(' ')
+
+    return (
+      // For screens below sm, we want to display them as rows
+      <div className="flex flex-col gap-6 sm:grid" style={{ gridTemplateColumns }}>
+        {children}
+      </div>
+    )
+  }
+
+  return <>{children}</>
+}
+
 /**
  * Our custom implementation of https://github.com/rjsf-team/react-jsonschema-form/blob/main/packages/core/src/components/templates/ObjectFieldTemplate.tsx
  * This allows us to provide specific UI options for styling the template (e.g. columns).
  * This implementation removes titles and descriptions from objects. It might be needed to add it back.
```

Comment:
so, this is now inacurate and we always display the title when it's provided in object, and don't display it if it's not .. right ?

Reply:
I clarified the comment.